lib/sacastats_web/templates/character/template.html.heex

```diff
@@ -52,7 +52,7 @@
 				<p class="d-inline">Stats last updated on: </p>
 			</div>
 			<div class="col-12 col-md-auto pl-3">
-				<strong><p class="d-inline date-time">
```

Comment:
Why was this removed?

Reply:
because `prettify_timestamp/2` adds an HTML element with the `date-time` class, so it's not needed here (and I think it was interfering with the date formatting iirc)